.github/workflows/ci.yml

```diff
@@ -283,18 +282,13 @@ jobs:
         run: |
           yarn install
           touch ~/.profile
-          mkdir "$HOME/.ssh"
-          mkdir "$HOME/.sbt"
-          mkdir "$HOME/.sbt/gpg"
```

Comment:
It's seems that this and also the couple things that I removed down below are all covered by sbt-ci-release, so there shouldn't be any reason to manually do this anymore. We won't really be able to tell until we just try.